src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -88,7 +88,10 @@ Also keep in mind, by default we disable source map upload in `dev` mode when ru
 
 ## Configure sentry-cli
 
-Under the hood, the `SentryWebpackPlugin` uses `sentry-cli`, which can be configured in one of two ways - using a properties file, or with environment variables - both of which are discussed below. For full details, see [the CLI configuration docs](https://docs.sentry.io/product/cli/configuration/). (If you choose to combine the two approaches, the environment variables will take precedence over values set in the properties file.)
+Under the hood, the `SentryWebpackPlugin` uses `sentry-cli`, which can be configured in one of two ways - using a properties file, or with environment variables - both of which are discussed below. For full details, see [the CLI configuration docs](https://docs.sentry.io/product/cli/configuration/).
```

Comment:
"Under the hood" is such an English-specific idiom that I think we should avoid it (I know this wasn't part of your edits, but I figure let's fix this now). Wondering if this text loses anything if we just take it out? 

```suggestion
The `SentryWebpackPlugin` uses `sentry-cli`, which can be configured in one of two ways - using a properties file, or with environment variables - both of which are discussed below. For full details, see [the CLI configuration docs](https://docs.sentry.io/product/cli/configuration/).
```

Reply:
Since I don't think so, applying your suggestion.